tsdb/db_test.go

```diff
@@ -3769,7 +3769,7 @@ func TestQuerierShouldNotFailIfOOOCompactionOccursAfterRetrievingQuerier(t *test
 		defer compactionComplete.Store(true)
 
 		require.NoError(t, db.CompactOOOHead(ctx))
-		require.Equal(t, float64(1), prom_testutil.ToFloat64(db.Head().metrics.chunksRemoved))
+		require.InDelta(t, float64(1), prom_testutil.ToFloat64(db.Head().metrics.chunksRemoved), 0.01)
```

Comment:
Remove the float64 conversion here

But also, the code could use chunksRemoved directly no ?

So Equal 1 would make sense to me